CONTRIBUTING.rst

```diff
@@ -196,7 +196,7 @@ Running System Tests
    > --key-file=$P12_CREDENTIALS_FILE
 
    # Create the indexes
-   $ gcloud preview datastore create-indexes system_tests/data/ \
+   $ gcloud preview datastore create-indexes system_tests/data/index.yaml \
```

Comment:
@dhermes This looks correct to me, per the [docs for `create-indexes`](https://cloud.google.com/sdk/gcloud/reference/preview/datastore/create-indexes).